metadata/nz_building_outlines_lifecycle.xml

```diff
@@ -98,17 +98,18 @@ This table provides the relationships between building outlines within mainland
 
 A building outline is a 2D representation of the roof outline of a building which has been classified from LINZ aerial imagery using a combination of automated and manual processes to extract and refine a building roof outline.
 
-Data vintage
-This dataset does not include building outline geometry, but provides the linkages between building_id's from different aerial imagery sources in the [NZ Building Outlines (All Sources)] (https://data.linz.govt.nz/layer/101292) dataset when building outlines change from single outlines split into multiple outlines, and from multiple outlines merged as single outlines.
+**Data vintage**
+

```

Comment:
I don't know why this space looks so huge? is it just cos of the diff check?

Reply:
Hmm. When I open it in Sublime I don't see spaces.